README-sandbox.md

```diff
@@ -44,7 +47,8 @@ security of mainnet nodes, though, and that's why they're mandatory.
 $ mina libp2p generate-keypair --privkey-path keys/node.key
 ```
 
-This command creates a key for the node operator. This is a key to use
+This command (which needs to be run from the mina repo) creates a key
+for the node operator. This is a key to use
```

Comment:
This is imprecise. What matters is that user has certain environment variables set, which usually Nix sets up for us. In your case `direnv` boils it down to being in a particular directory, but for someone who is not using `direnv` it won't solve anything.

```suggestion
This command (which depends on environment variables set up by Nix)
creates a key for the node operator. This is a key to use
```
This way reader has better understanding of what's going on and even in the unlikely event of  not using Nix at all they'll be able to find a solution that suits their needs.